packages/wrangler/src/pubsub/pubsub-commands.tsx

```diff
@@ -617,6 +617,139 @@ export function pubSubCommands(
 				}
 			);
 
+			brokersYargs.command(
+				"list-subscribers",
```

Comment:
Something to consider, normally in wrangler2 we'd name the command `subscribers`, then you'd have `list`, `add`, `remove`, and `update` as easily discoverable options to the subscribers command

Reply:
Would also appreciate tests 😃

Reply:
@rozenmd That would then be:

```sh
pubsub broker subscribers add <BROKER_NAME> --script-name=my-subscriber-worker --filter="metrics/"
```

... which feels a bit unwieldy, especially if we retain the positional argument for `BROKER_NAME`. A subscriber is scoped to a broker. You could have:

```sh
pubsub broker subscribers add --script-name=my-subscriber-worker --filter="metrics/" --broker-name=BROKER_NAME
```

... but that now diverges from the positional `BROKER_NAME` arg used everywhere else within the `broker` sub-command hierarchy.

Note:

1. `broker` is singular as that was the style enforced when we originally committed this
2. Should `subscribers` then be plural or singular? (has our style changed since?)
3. Open to other suggestions here.

@JacobMGEvans - this is just a WIP. Tests coming later.

Reply:
cc/ @rozenmd if you can take a look at my questions above